frontend/src/theme/components/Tabs.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  ChakraTheme,
+  ComponentMultiStyleConfig,
+  SystemStyleObjectRecord,
+  ThemingPropsThunk,
+} from '@chakra-ui/react'
+import { merge } from 'lodash'
+
+const variantLine: ThemingPropsThunk<SystemStyleObjectRecord, ChakraTheme> = (
+  props,
+) => {
+  const { orientation } = props
+  const isVertical = orientation === 'vertical'
+  const marginProp = isVertical ? 'marginStart' : 'marginBottom'
+  return {
+    root: {
+      w: 'auto',
+      p: '24px',
+      mx: '-16px', // ignore parent padding and extend full width of screen
+    },
+    tablist: {
+      overflowX: 'scroll',
+      borderColor: 'transparent',
+      /* Hide scrollbar for IE and Edge */
+      '&::MsOverflowStyle': 'none',
+      /* Hide scrollbar for Firefox */
+      '&::scrollbarWidth': 'none',
+      /* Hide scrollbar for Chrome, Safari and Opera */
+      '&::-webkit-scrollbar': {
+        display: 'none',
+      },
+    },
+    tabPanel: {
+      p: '4px',
+    },
+    tab: {
+      textTransform: 'uppercase',
+      p: '4px',
+      [marginProp]: '4px',
+      _focus: {
+        boxShadow: 'none',
+      },
+    },
+  }
+}
+
+const variantLight: ThemingPropsThunk<SystemStyleObjectRecord, ChakraTheme> = (
+  props,
+) => {
+  return merge(variantLine(props), {
+    root: {
+      bg: 'white',
+    },
+    tab: {
+      _hover: {
+        color: 'primary.400',
+      },
+      _selected: {
+        color: 'primary.500',
+      },
+      _active: {
+        bg: 'white',
+      },
+      transitionProperty: 'common',
+      color: 'primary.300',
+    },
+  })
+}
+
+const variantDark: ThemingPropsThunk<SystemStyleObjectRecord, ChakraTheme> = (
+  props,
+) => {
+  return merge(variantLine(props), {
+    root: {
+      bg: 'secondary.500',
+    },
+    tab: {
+      _active: {
+        bg: 'secondary.500',
+      },
+      _hover: {
+        color: 'secondary.200',
+      },
+      _selected: {
+        color: 'white',
+      },
+      transitionProperty: 'common',
+      color: 'secondary.300',
+    },
+  })
+}
+
+export const Tabs: ComponentMultiStyleConfig = {
+  parts: [],
+  baseStyle: {
+    tab: {
+      mr: '6%',
```

Comment:
This seems arbitrary, is it really what the design is?

Reply:
I made an estimation using the distance between the tabs and the total length of the `_IndividualTabs` component in figma. I realised this value is wrong, it should be around '15%'. Is it okay to use this estimation?

Reply:
looking at the figma design, the desktop gap between tab items is 160px and the mobile gap is 32px. you can check the distance between components by selecting the component, then holding the cmd key.

do double check with @pearlyong on the real gap and if it changes